tsdb/engine/tsm1/engine_cursor.go

```diff
@@ -23,6 +24,10 @@ func (e *Engine) CreateCursor(ctx context.Context, r *tsdb.CursorRequest) (tsdb.
 		return nil, nil
 	}
 
+	if grp := metrics.GroupFromContext(ctx); grp != nil {
+		grp.GetCounter(numberOfRefCursorsCounter).Add(1)
```

Comment:
Whats this?

Reply:
the number of TSM cursors that have been created; it is ultimately serialized as a tag